docs/data/base/components/trap-focus/ContainedToggleTrappedFocus.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Stack from '@mui/material/Stack';
+import TrapFocus from '@mui/base/TrapFocus';
+
+export default function BasicTrapFocus() {
+  const [open, setOpen] = React.useState(false);
+
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        alignItems: 'center',
+        flexDirection: 'column',
+      }}
+    >
```

Comment:
Unnecessary, we can simplify the demo without

```suggestion
```

Reply:
I was going for consistency with the other demos

e.g. [DisableEnforceFocus](https://github.com/mui/material-ui/blob/master/docs/data/base/components/trap-focus/DisableEnforceFocus.js), [LazyTrapFocus](https://github.com/mui/material-ui/blob/master/docs/data/base/components/trap-focus/LazyTrapFocus.js), [PortalTrapFocus](https://github.com/mui/material-ui/blob/master/docs/data/base/components/trap-focus/PortalTrapFocus.js)

Would you suggest that they all be changed?

Reply:
Ah, nevermind I went to do that and then immediately saw why it's necessary in those cases and not this one!